packages/edit-post/src/components/sidebar/plugin-post-excerpt/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
If you decide to abandon the snapshot testing approach, let's not forget to get rid of that file.